sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -52,6 +52,8 @@ export interface ServiceBusClientOptions {
  * Describes the client that allows interacting with a Service Bus instance.
  * Holds the AMQP connection to the Service Bus Namespace and is the entry point for using Queues,
  * Topics and Subscriptions.
+ * Each message operation on a different entity involves setting up a new AMQP `LinkEntity`.
+ * There is a limit of a maximum of 1000 such operations to happen simultaneously on a single AMQP connection.
```

Comment:
Is there a public doc that describes this limitation?

Reply:
https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-performance-improvements#queue-with-a-large-number-of-senders

But it seems misleading as it says 5000 for AMQP

Reply:
If the service public doc is misleading, we should create a PR to fix that document.
The sdk need not document service side capabilities/limitations if it is already done in a public document

Reply:
Ok, closing this issue and PR